tests/density.js

```diff
@@ -2,22 +2,121 @@ import http from 'k6/http';
 import { check } from 'k6';
 import { Kubernetes } from 'k6/x/kubernetes';
 import * as deploy from "./common/common.js";
-import { Gauge, Counter } from 'k6/metrics';
+import { Gauge } from 'k6/metrics';
 import exec from 'k6/execution';
 import { sleep } from 'k6';
 
-export let options = {
-  stages: [
-    { target: 20, duration: '10s' },
-    { target: 0, duration: '10s' }
-  ],
-  setupTimeout: '300s',
-  noConnectionReuse: true
-};
-
+const testName = "density";
+const route = ""
+const replicas = 1;
+const namespace = `${__ENV.NAMESPACE}` != "undefined" ? `${__ENV.NAMESPACE}` : "default";
+const executor = `${__ENV.EXECUTOR}` != "undefined" ? `${__ENV.EXECUTOR}` : "containerd-shim-spin";
+const repo = `${__ENV.REPO}` != "undefined" ? `${__ENV.REPO}` : "rg.fr-par.scw.cloud/dlancshire-public/template-app-";
+const tag = `${__ENV.TAG}` != "undefined" ? `${__ENV.TAG}` : "latest";
 const timeToDeployTenApps = new Gauge('time_to_deploy_ten_apps');
 const deployedApps = new Gauge('deployed_apps');
-const waitAppsReadyTimeout = 300;
+const maxScenarioDurationSecs = `${__ENV.MAX_SCENARIO_DURATION_SECS}` != "undefined" ? `${__ENV.MAX_SCENARIO_DURATION_SECS}` : 180;
+const restIntervalSecs = `${__ENV.REST_INTERVAL_SECONDS}` != "undefined" ? `${__ENV.REST_INTERVAL_SECONDS}` : 10;
+
+// Should match the number of apps deployed by the last stage
+// and be divisible by the batch size
+// TODO: make this configurable
+const totalApps = 50;
+
+export let options = {
+  tags: {
+    test: testName,
+  },
+  noConnectionReuse: true,
+  discardResponseBodies: true,
+  setupTimeout: '120s',
```

Comment:
```suggestion
  setupTimeout: '300s',
```
In my testing, it looks like 120s isn't quite sufficient, at least on the current default `Standard_A2_v2` SKU.

Reply:
Setup is not longer deploying apps in this test. I am going to remove the `setupTimeout`. Rather it is `maxScenarioFurationSecs` that we need to adjust. The problem with making this 600s it that it will make this test 30000s long since for the scenarios to not overlap we have to set `startTime`. This may be a reason to bail on these changes

Reply:
Ah ok 👍

Reply:
Just to clarify, i meant bail on entire changes to the density test. Maybe we do 300s and see how fast it is and file the issue if it keeps failing

Reply:
Ohh, gotcha.  Defer to you here.  Can certainly see how this works and address as needed...